cv32e40s/tests/cfg/default.yaml

```diff
@@ -9,6 +9,70 @@ ovpsim: >
     --override cpu/hpmcounter_undefined=T
     --override cpu/PMP_registers=64
     --override cpu/PMP_grain=0
+    --override cpu/mask_pmpaddr0=0x00000000
```

Comment:
Is there really no PMP_NUM_REGIONS equivalent that could simplify this commit?

Reply:
Negative, this has to be specified, either here or in sv - adding it here as a quickfix